corehq/apps/receiverwrapper/rate_limiter.py

```diff
@@ -0,0 +1,30 @@
+from corehq.project_limits.rate_limiter import RateDefinition, RateLimiter, \
+    PerUserRateDefinition
+
+# Danny promised in an Aug 2019 email not to enforce limits that were lower than this.
+# If we as a team end up regretting this decision, we'll have to reset expectations
+# with the Dimagi NDoH team.
+rates_promised_not_to_go_lower_than = RateDefinition(
+    per_week=115,
+    per_day=23,
+    per_hour=3,
+    per_minute=0.07,
+    per_second=0.005,
+)
+
+floor_for_small_domain = RateDefinition(
+    per_day=50,
+    per_hour=30,
+    per_minute=10,
+    per_second=1,
+)
+
+test_rates = PerUserRateDefinition(
+    per_user_rate_definition=rates_promised_not_to_go_lower_than.times(2.0),
```

Comment:
why `times(2.0)`?

Reply:
I wanted to try an approach of starting out high and rate limiting only extreme outliers before pulling it in and landing somewhere comfortable. In order to get a good sense of where to end up I wanted to start at the high end of the range. What I don't want to do is to kind of slam on new restrictions to our users and have everyone scramble to adjust to them. (I realize this PR doesn't actually rate limit yet, but I wanted to collect data for the place I was considering starting rate limiting at.)

Does that approach make sense?